sympy/geometry/line.py

```diff
@@ -1083,24 +1093,68 @@ class Line(LinearEntity):
     >>> s = Segment((0, 0), (0, 1))
     >>> Line(s).equation()
     x
+
+    >>> # a circle object is returned
```

Comment:
this still needs to be corrected: remove ref to circle and make non-inline

Reply:
@smichr Done that now. Please take a look.